eng/scripts/ci-source-build.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Is this file identical to the one in Extensions?

Reply:
Not quite, I had to add a step to build the RepoTask.csproj.